### PR TITLE
Release notes for 0.19.0.

### PR DESCRIPTION
**Pull request by wRAR**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,22 @@
...
+===================
+
+* Now requires ``attrs >= 22.2.0``.
+
+* Added the :class:`~.DropLowProbabilityItemPipeline` that drops items with the
+  ``probability`` value lower than a set threshold.
+
+* Added the :attr:`~.ListMetadata.validationMessages` attribute.
+
+* Reorganized the module structure.
```
Comment (wRAR) [inline CHANGELOG.rst:15]: Should this be more verbose?

Comment (Gallaecio) [reply CHANGELOG.rst:15]: Since most (all?) changes are internal, this is probably OK.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -2,6 +2,22 @@
...
+* Added :class:`~.pipelines.DropLowProbabilityItemPipeline` that drops items
+  with the ``probability`` value lower than a set threshold.
+
+* Added the :attr:`.ListMetadata.validationMessages` attribute.
+
+* Reorganized the module structure.
+
+* Documentation improvements.
+
+* Test and CI fixes and improvements.
```
Comment (kmike) [inline CHANGELOG.rst:19]: There is a sneaky change in https://github.com/zytedata/zyte-common-items/pull/90/: new ListMetadata.get_date_downloaded_parsed method

File: CHANGELOG.rst (hunk 1 above)
Comment (kmike) [reply CHANGELOG.rst:15]: There are also some classes which are promoted to public, e.g. BaseMetadata, ListMetadata, DetailsMetadata

Comment (kmike) [reply CHANGELOG.rst:24]: We may also mention zyte_common_items.converters module; the code was scattered in various places before, but now it contains useful attrs convertors in a single place

File: CHANGELOG.rst (hunk 2 above)
Comment (wRAR) [reply CHANGELOG.rst:28]: I've noticed it, but wasn't sure if it's intended to be used outside tests.

File: docs/reference/converters.rst [hunk 3]
```diff
@@ -0,0 +1,6 @@
+==========
+Converters
+==========
+
+.. automodule:: zyte_common_items.converters
```
Comment (wRAR) [inline docs/reference/converters.rst:5]: Getting `WARNING: py:class reference target not found: web_poet.page_inputs.url._Url` here, but as it's indeed not mentioned in the web-poet docs I guess there is nothing to do?

File: CHANGELOG.rst (hunk 1 above)
Comment (kmike) [reply CHANGELOG.rst:29]: Just checked it; it seems that's only these 3 classes :)

File: docs/reference/converters.rst (hunk 3 above)
Comment (kmike) [reply docs/reference/converters.rst:5]: I think that's ok, if it doesn't break CI

**Pull request merged**